docs/UserGuide.md

```diff
@@ -63,19 +65,6 @@ Covigent is a desktop app for managing information of quarantined individuals an
   
 </div>
 
-### Viewing help : `help`
-
-Shows a message explaning how to access the help page.
-
-![help message](images/helpMessage.png)
-
-Format: `help`
-
-
-
-
```

Comment:
Can delete white space, seems unnecessary.

Reply:
the - means that its deleted